### PR TITLE
MEN-3775 enable wifi on startup

### PR DESCRIPTION
**Pull request by merlin-northern**

### CODE REVIEW

File: 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md [hunk 1]
```diff
@@ -140,20 +140,31 @@ Gateway=$IP_OF_MENDER_SERVER_FROM_DEVICE
...
+	psk="YourPassword"
+}
+```
+where YourCountry stands for two-letter country code (e.g.: PL for Poland),
+YourSSID, and YourSSID are the SSID of your network and the password, respecively.
+And then run:
 
 ```bash
-mender-artifact cat "$MENDER_IMGPATH":/etc/wpa_supplicant/wpa_supplicant-nl80211-wlan0.conf | sed "s#psk=\"password\"#psk=\"$NW_PASSWORD\"#" | sed "s#ssid=\"ssid\"#ssid=\"$NW_SSID\"#" > tmpf; mender-artifact cp tmpf "$MENDER_IMGPATH":/etc/wpa_supplicant/wpa_supplicant-nl80211-wlan0.conf && rm tmpf
+mender-artifact cp wpa.in "$MENDER_IMGPATH":/boot/wpa_supplicant.conf
```
Comment (kacf) [inline 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md:163]: Is `/boot` really correct here? Shouldn't it be `/etc/wpa_supplicant`?

File: 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md [hunk 2]
```diff
@@ -140,20 +140,31 @@ Gateway=$IP_OF_MENDER_SERVER_FROM_DEVICE
...
 ```
 
-should have your wpa configuration set up correctly on start up.
+Now you should have wpa configuration set up correctly on start up. (Do not forget
+to remove `wpa.in` file, it contains your password).
```
Comment (kacf) [inline 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md:167]: Good thinking!

File: 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md (hunk 1 above)
Comment (merlin-northern) [reply 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md:163]: yes, there is this service in place:

```
root@raspberrypi:~# cat /lib/systemd/system/raspberrypi-net-mods.service
[Unit]
Description=Copy user wpa_supplicant.conf
ConditionPathExists=/boot/wpa_supplicant.conf
Before=dhcpcd.service
After=systemd-rfkill.service

[Service]
Type=oneshot
RemainAfterExit=yes
ExecStart=/bin/mv /boot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf
ExecStartPost=/bin/chmod 600 /etc/wpa_supplicant/wpa_supplicant.conf
ExecStartPost=/usr/sbin/rfkill unblock wifi

[Install]
WantedBy=multi-user.target
```

**Comment by merlin-northern** [comment]
> Does this section exist also in `master`?

nope, and also does not exist on hosted.

File: 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md (hunk 1 above)
Comment (kacf) [reply 01.Getting-started/02.On-premise-installation/04.Deploy-a-system-update-demo/docs.md:163]: Aha!

**Comment by merlin-northern** [comment]
thanks @kacf !

**Pull request merged**